extensions/wikidata/src/org/openrefine/wikidata/schema/WbItemDocumentExpr.java

```diff
@@ -96,12 +96,12 @@ public ItemUpdate evaluate(ExpressionContext ctxt)
 
     @JsonProperty("nameDescs")
     public List<WbNameDescExpr> getNameDescs() {
-        return nameDescs;
+        return nameDescs.stream().collect(ImmutableList.toImmutableList());
```

Comment:
`Collections.unmodifiableList()` is probably a simpler (and higher performance?) idiom here.
```suggestion
        return Collections.unmodifiableList(nameDescs);
```